_data/simple-icons.json

```diff
@@ -6697,9 +6697,14 @@
         },
         {
             "title": "Veeam",
-            "hex": "00B336",
+            "hex": "FF008E",
             "source": "https://www.veeam.com/newsroom/veeam-graphics.html"
         },
+        {
+            "title": "Veepee",
+            "hex": "00B336",
+            "source": "http://www.veepee.com/"
+        },
```

Comment:
I think you have changed the color at the wrong place in `simple-icons.json`.
Also, the color used in the logo and in the website is `#EC008C` (https://static.preprod.vente-privee.com/vp5/_web/site/vp/_common/img/00_int_door_logoBlack.svg).